shared/api/models/EventModel.ts

```diff
@@ -14,6 +14,7 @@ class EventModel extends ExtendedPageModel {
   _location: LocationModel<number | null> | null
   _excerpt: string
   _featuredImage: FeaturedImageModel | null
+  _locationPath: string | null
```

Comment:
:wrench: See above, the naming should be consistent with the rest of our codebase:
```suggestion
  _poiPath: string | null
```